packages/workshop-utils/src/epic-api.server.ts

```diff
@@ -70,10 +71,20 @@ export async function getEpicVideoInfos(
 	epicWebUrls?: Array<string> | null,
 	{ request, timings }: { request?: Request; timings?: Timings } = {},
 ) {
-	if (!epicWebUrls) return {}
+	const log = logger('epic-api:video-infos')
```

Comment:
Move this declaration to the module-level so it can be shared across all functions of the module.